### PR TITLE
Remove "address" from user object in read only values

### PR DESCRIPTION
**Pull request by adamgall**
This PR only does one thing, removes the `address` property from the `user` object in the `readOnlyValues` state.

### CODE REVIEW

File: src/providers/App/useReadOnlyValues.ts [hunk 1]
```diff
@@ -67,7 +71,7 @@ export const useReadOnlyValues = (
       }
     };
 
-    const address = _account;
+    const address = account;
     Sentry.setUser(address ? { id: address } : null);
```
[the remark this answers is not in the document]
Comment (mudrila) [reply src/providers/App/useReadOnlyValues.ts:75]: Just thinking out loud - no actions for this PR, ser

[the remark this answers is not in the document]
Comment (adamgall) [reply src/providers/App/useReadOnlyValues.ts:75]: Definitely will, that's coming up soon. I'm dismantling this thing one step at a time, hence the multiple PRs.

File: src/components/DaoCreator/StepButtons.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import { Flex, Button, Icon } from '@chakra-ui/react';
-import { CaretRight, CaretLeft } from '@phosphor-icons/react';
+import { Button, Flex, Icon } from '@chakra-ui/react';
```
Comment (DarksightKellar) [inline src/components/DaoCreator/StepButtons.tsx:1]: Someone's IDE's been fighting with these import orders lol. This ascending alphabetical order is preferred imo. @decentdao/engineering we should make sure our setups match

**Review by DarksightKellar — APPROVED** [review]
Yes.

File: src/components/DaoCreator/StepButtons.tsx (hunk 2 above)
Comment (adamgall) [reply src/components/DaoCreator/StepButtons.tsx:1]: You wanna try to figure that out? Should be enforceable with a prettier or lint rule or something

**Pull request merged**